src/library/unfold_macros.cpp

```diff
@@ -119,8 +119,6 @@ class unfold_untrusted_macros_fn {
 };
 
 static bool contains_untrusted_macro(unsigned trust_lvl, expr const & e) {
-    if (trust_lvl > LEAN_BELIEVER_TRUST_LEVEL)
-        return false;
     return static_cast<bool>(find(e, [&](expr const & e, unsigned) {
```

Comment:
This was a performance optimization. 
`contains_untrusted_macro` is invoked when importing files. 
When we are importing hundreds of files, the following `find` consumes a significant amount of time. At affects the `flycheck` response time.
The idea of the statement above was: if the trust level is high enough trust all macros.